src/core/service.factory.ts

```
@@ -0,0 +1,204 @@
+   */
+  static getStatusListService(): StatusListService | null {
+    return ServiceFactory.statusListService;
+  }
+
+  /**
+   * Gets the current AssertionService instance (if available)
+   */
+  static getAssertionService(): AssertionService | null {
+    return ServiceFactory.assertionService;
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)